packages/ra-ui-materialui/src/input/RadioButtonGroupInput.js

```diff
@@ -78,70 +76,41 @@ const styles = createStyles({
  * The object passed as `options` props is passed to the material-ui <RadioButtonGroup> component
  */
 export const RadioButtonGroupInput = ({
-    classes,
-    className,
-    label,
-    resource,
-    source,
-    input,
-    isRequired,
     choices,
-    options,
-    meta,
     helperText,
+    label,
+    onBlur,
+    onChange,
+    onFocus,
+    options,
     optionText,
     optionValue,
+    resource,
+    source,
     translateChoice,
+    validate,
     ...rest
 }) => {
-    const translate = useTranslate();
+    const classes = useStyles({});
```

Comment:
I tend to let a way for developers to override the styles, by accepting a `classes` that I pass to `useStyles` at runtime 

See example at https://github.com/marmelab/react-admin/blob/8cdd8cedcf90967404c7e6d71a3403c3a35f8a1c/packages/ra-ui-materialui/src/button/BulkDeleteWithConfirmButton.js#L42-L54

Reply:
I'll probably have to check all previously migrated components for this